pimd/pim_iface.c

```diff
@@ -1235,6 +1245,24 @@ static struct gm_join *gm_join_find(struct list *join_list, pim_addr group_addr,
 	return 0;
 }
 
+static struct static_group *static_group_find(struct list *static_group_list,
+					      pim_addr group_addr,
+					      pim_addr source_addr)
+{
+	struct listnode *node;
+	struct static_group *stgrp;
+
+	assert(static_group_list);
+
+	for (ALL_LIST_ELEMENTS_RO(static_group_list, node, stgrp)) {
+		if ((!pim_addr_cmp(group_addr, stgrp->group_addr)) &&
+		    (!pim_addr_cmp(source_addr, stgrp->source_addr)))
+			return stgrp;
+	}
+
+	return 0;
```

Comment:
blech.  0?  Let's return NULL.  NULL is not guaranteed to be 0 by the standard.  we should avoid this pattern at all costs